src/server/app.js

```diff
@@ -1,22 +1,26 @@
 import express from 'express';
 import mongoose from './util/mongoose.util.js';
+import morgan from 'morgan';
 import graphqlHTTP from 'express-graphql';
 import schema from './graphQLschema/index';
 import jwtUtil from './util/jwt.util';
+import bodyParser from 'body-parser';
 
 const app = express();
 
 const PORT = process.env.PORT;
 
 mongoose.connect();
+app.use(morgan('dev'));
+app.use(bodyParser.json());
 
 app.post('/graphql', jwtUtil.apiProtector, graphqlHTTP((request) => {
   const startTime = Date.now();
   return {
     schema: schema,
     graphiql: true,
     rootValue: { request },
-    extensions() {
+    extensions(ext) {
```

Comment:
it is in graphql-express check this out.
[express-graphql](https://github.com/graphql/express-graphql)